pyof/v0x01/controller2switch/packet_out.py

```diff
@@ -112,13 +112,14 @@ def _update_actions_len(self):
             self.actions_len = ListOfActions(self.actions).get_size()
 
     def _validate_in_port(self):
-        port = self.in_port
-        valid = True
-        if isinstance(port, Port):
-            if port not in _VIRT_IN_PORTS:
-                valid = False
-        elif isinstance(port, int) and (port < 1 or port >=
-                                        Port.OFPP_MAX.value):
-            valid = False
-        if not valid:
-            raise ValidationError('{} is not a valid input port.'.format(port))
+        """Validate in_port attribute.
+
+        Raises:
+            ValidationError: If in_port is a invalid port.
+
+        """
+        is_valid_range = self.in_port > 0 and self.in_port <= Port.OFPP_MAX
```

Comment:
@renanrodrigo, me and @macartur think that `Port.OFPP_MAX` should be a valid port value. Do you agree?

Reply:
Agreed. I do think it's valid.

Reply:
Tks!